src/libmongoc/tests/test-mongoc-client-side-encryption.c

```diff
@@ -6246,6 +6281,8 @@ test_auto_datakeys (void *unused)
 static void
 _do_cec_test (void (*test) (const char *kmsProvider))
 {
+   SKIP_IF_NO_QUERYABLE_ENCRYPTION_V2;
```

Comment:
Suggest moving this into the `CEC_TEST` uses below for consistency in test function naming.